### ISSUE TITLE
Caption issue 

### ISSUE THREAD

**Issue by salmaelgayar** [pritishmishra703/Image-Captioning#1]
Hello 
Thank you for sharing the amazing code!
I tried to run this code on different X-ray data sets for medical captioning.
Unfortunately, I always get the same word for caption: "the the the..."
![Untitled](https://github.com/pritishmishra703/Image-Captioning/assets/32023822/d02b5dd8-3c37-4cfe-98dd-260368348bf0)
 for any input image
Do you have any suggestions as to why this happened?

**Comment by pritishmishra703** [comment]
The model is not fine-tuned to handle X-ray Images. It would require further training (on X-ray images) to be able to do so.

**Comment by salmaelgayar** [comment]
Thank you for your reply. Actually, I solved the problem and did what you said and trained the medical x-rays to the model.
but I couldn't reach high accuracy with the validation dataset.
Any recommendations for the best fine-tuned neural network layer to be added to this architecture to get the best caption validation results?!
